style.css

```diff
@@ -0,0 +1,226 @@
+@import url("https://fonts.googleapis.com/css2?family=Alumni+Sans+Collegiate+One&family=Montserrat:wght@100;500&display=swap");
+@import url("https://fonts.googleapis.com/css2?family=Alumni+Sans+Collegiate+One&family=Crete+Round&family=Montserrat:wght@100;500&display=swap");
+
+* {
+  margin: 0;
+
+  /* border: 1px solid black; */
+}
+
+/* class to import */
+.white {
+  color: white;
+}
+
+.blue-1 {
+  color: #091e42;
+}
+
+.blue-2 {
+  color: #091e42;
+}
+
+.yelow {
+  color: #ff6b00;
+}
+
+.bg-btn {
+  background-color: rgba(255, 255, 255, 0.24);
+}
+
+.font-montesori {
+  font-family: "Montserrat", sans-serif;
+}
+
+/* Content of the page */
+.header-container {
+  display: flex;
+  width: auto;
+  height: auto;
+  align-items: center;
+  background-color: #3c3a39;
+  justify-content: space-between;
+  padding: 20px;
+  position: sticky;
+  z-index: 1;
+  top: 0;
+  font-family: "Montserrat", sans-serif;
+}
+
+.headline-container {
+  background-color: #1c1c19;
+  background-image: url(./assets/images/Illustration.png);
+  background-position: right bottom;
+  background-repeat: no-repeat;
+  display: flex;
+  flex-direction: column;
+  padding: 100px 20px 20px 20px;
+  height: 60vh;
+  gap: 10px;
+  position: relative;
+  font-family: "Montserrat", sans-serif;
+}
+
+.headline-title {
+  color: #ff6b00;
+  font-size: 50px;
+  font-style: unset;
+  font-family: "Crete Round", serif;
+}
+
+.social-media-icones-container {
+  display: flex;
+}
+
+/* today */
+#works-container {
+  display: grid;
+  grid-template-rows: 100px auto 450px 450px 450px 100px;
+}
+
+.recent-works-title-block {
+  display: flex;
+  align-items: center;
+  justify-content: center;
+  font-family: "Crete Round", serif;
+  font-size: 30px;
+}
+
+.recent-works {
+  display: flex;
+  flex-direction: column;
+  padding: 0 20px 20px 20px;
+  gap: 15px;
+}
+
+.tools-container {
+  display: flex;
+  gap: 20px;
+}
+
+.tools {
+  padding: 10px;
+  border: 1px solid #8993a4;
+  list-style: none;
+}
+
+.see-project-btn {
+  background-color: #ff6b00;
```

Comment:
- [ ] Please add a hover and pressed state to all your buttons as indicated in the figma template.